examples/ai_safety/test_ai_safety.py

```
@@ -0,0 +1,159 @@
+        if not process_code:
+            print(f"✗ FAILED: No PROCESS block found")
+            return {'program': program_name, 'success': False, 'error': 'No PROCESS block'}
+        
+        print(f"\nProcess Code Preview: {process_code[:80]}...")
+        
+        # Create interpreter and context
+        interpreter = PhiPiEInterpreterFixed()
+        context = FieldContext()
+        
+        # Execute
+        print(f"\nExecuting...")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)